content/de/community/_index.html

```diff
@@ -4,58 +4,253 @@
-            in allen Interaktionen durch. Wenn Sie einen Verstoß gegen den Verhaltenskodex bei einer Veranstaltung oder Sitzung,
-            in Slack oder in einem anderen Kommunikationsmechanismus feststellen, wenden Sie sich
-            bitte an das <a href="https://github.com/kubernetes/community/tree/master/committee-code-of-conduct">Kubernetes Code of Conduct Committee</a> <a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a>. Ihre Anonymität wird geschützt.
-          </p>
-         </div>
-    </main>
-</section>
-
-<section id="talkToUs">
-    <main>
-        <h3>Talk to Us!</h3>
-        <h4>Wir würden uns freuen, von Ihnen zu hören, wie Sie Kubernetes verwenden<br>und was wir tun können, um es besser zu machen.</h4>
-        <div id="bigSocial">
-            <div>
-                <a href="https://twitter.com/kubernetesio">@kubernetesio</a>
-                <p>Erhalten Sie die neuesten Nachrichten und Updates.</p>
-            </div>
-            <div>
-                <a href="https://github.com/kubernetes/kubernetes">Github Project</a>
-                <p>Informieren Sie sich über das Projekt und erwägen Sie, einen Beitrag zu leisten.</p>
-            </div>
-            <div>
-                <a href="http://slack.k8s.io/">#kubernetes-users</a>
-                <p>Unser Slack-Kanal ist der beste Weg, um unsere Ingenieure zu kontaktieren und Ihre Ideen mit ihnen zu teilen.</p>
-            </div>
-            <div>
-                <a href="http://stackoverflow.com/questions/tagged/kubernetes">Stack Overflow</a>
-                <p>Unser Benutzerforum ist ein großartiger Ort, um Community-Support zu erhalten.</p>
-            </div>
-        </div>
-    </main>
-</section>
+<div class="newcommunitywrapper">
+<div class="banner1">
+  <img src="/images/community/kubernetes-community-final-02.jpg" alt="Kubernetes Conference Gallery" style="width:100%;padding-left:0px" class="desktop">
+  <img src="/images/community/kubernetes-community-02-mobile.jpg" alt="Kubernetes Conference Gallery" style="width:100%;padding-left:0px" class="mobile">
+</div>
+
+<div class="intro">
+<br class="mobile">
+<p>Die Kubernetes-Community - Nutzer, Mitwirkende und die Kultur, die wir gemeinsam aufgebaut haben - ist einer der Hauptgründe für den kometenhaften Aufstieg dieses Open-Source-Projekts. Unsere Kultur und unsere Werte wachsen und entwickeln sich mit dem Wachstum und der Veränderung des Projekts selbst. Wir alle arbeiten gemeinsam an der ständigen Verbesserung des Projekts und der Art und Weise, wie wir daran arbeiten.
+<br><br>Wir sind die Leute, die Probleme und Pull-Requests einreichen, an SIG-Treffen, Kubernetes-Treffen und der KubeCon teilnehmen, sich für die Einführung und Innovation von Kubernetes einsetzen, <code>kubectl get pods</code> ausführen und auf tausend andere wichtige Arten beitragen. Lies weiter, um zu erfahren, wie Du dich engagieren und Teil dieser faszinierenden Gemeinschaft werden kannst.</p>
```

Comment:
I'd definitely consider adding the expansion of SIG (special interest group) - not everyone will know this abbreviation.